packages/synapse-react-client/src/components/AccessRequirementList/ManagedACTAccessRequirementRequestFlow/DataAccessRequestAccessorsFilesForm/DataAccessRequestAccessorsFilesForm.stories.tsx

```diff
@@ -82,10 +59,9 @@ export const Request: Story = {
 }
 export const Renewal: Story = {
   args: {
-    isAuthenticated: true,
     subjectId: MOCK_FOLDER_ID,
     subjectType: RestrictableObjectType.ENTITY,
-    managedACTAccessRequirement: mockManagedACTAccessRequirement,
+    managedACTAccessRequirement: mockManagedAccessRequirementWithNoACL,
```

Comment:
The data access request/renewal is fetched using the AR ID, so with the fixed handlers, we'll use a different mock AR to retrieve the renewal object.